CMakeLists.txt

```diff
@@ -108,6 +108,7 @@ option(UA_ENABLE_DISCOVERY "Enable Discovery Service (LDS)" ON)
 option(UA_ENABLE_JSON_ENCODING "Enable JSON encoding" ON)
 option(UA_ENABLE_XML_ENCODING "Enable XML encoding (EXPERIMENTAL)" OFF)
 option(UA_ENABLE_NODESETLOADER "Enable nodesetLoader public API" OFF)
+option(UA_ENABLE_PUSHMANAGEMENT "Enable pushManagement support" OFF)
```

Comment:
The build flag should contain the term "GDS".
For example `UA_ENABLE_GDS_PUSHMANAGEMENT`.
Can we differentiate the name for push-management between the GDS appliance and the receiving end?